dbqueries.py

```diff
@@ -71,5 +73,24 @@ class QueryDict(TypedDict):
                 'rows_to_ignore': []
             }
         }
+    },
+    'udp_learner_activity_updated': {
+        'output_file_name': 'udp_learner_activity_updated.csv',
+        'data_source': 'UDP_Context_Store',
+        'query_name': 'UDP Learner Activity Updated',
```

Comment:
Maybe add more readable query name here, like "UDP data freshness indicator: with last update date in UDP Learner Activity table"